.github/workflows/run.yml

```diff
@@ -0,0 +1,46 @@
+name: Run
+
+on:
+  workflow_dispatch:
+
+jobs:
+  packager:
+    name: Packager
+    runs-on: ubuntu-latest
+    strategy:
+      matrix: 
+        type:
+          - full
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.PAT }}
+          
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.1'
+          
+      - name: Get Composer Cache Directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+      - name: Cache PHP dependencies
+        uses: actions/cache@v2
+        id: actions-cache
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: ${{ runner.os }}-composer-
+          
+      - run: composer install --prefer-dist --no-dev
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - run: composer run build -- $REMOTE $PACKAGE --type=$TYPE
+        env:
+          REMOTE: ${{ secrets.REMOTE }}
+          PACKAGE: ${{ secrets.PACKAGE }}
```

Comment:
These don't need to be secrets right? Might as just hardcode them here, or set them as `inputs` in case we wanted to change them for a workflow run.

Reply:
This is the only drawback of using a git abstraction in php: you need to [specify a token inside the remote URI](https://github.com/czproject/git-php/tree/32505124ef2a8651f10a5afebace58f85eff2f93#remotes).
I've updated the workflow file to make it a bit more explicit.

Reply:
I see you broke out `TOKEN` which makes sense, but why is `PACKAGE_NAME` a secret? Wouldn't that just be `wordpress` in our case?

Reply:
@swalkinshaw Secrets are just a way to store variables without hard writing them into config files.
I think it is always better to let the opportunity to anyone to use code without having to rewrite the file, as far as possible.

Reply:
Yeah I was just thinking it could be a plain input vs a secret. The downside of an input is it needs to either be manually input on every run, or hardcode a default value.

Not a big deal either way 👍